sn_client/src/audit/spend_dag.rs

```diff
@@ -185,7 +188,7 @@ impl SpendDag {
         }
 
         // link to ancestors
-        let spend_amount = spend.token();
+        const PENDING_AMOUNT: NanoTokens = NanoTokens::from(0);
```

Comment:
we normally put `const` at head of the file ?

Reply:
I was wondering about using const or just a regular variable here, since it's only useful in this function and for the sake or readability I'd like to keep it as close to where it's used. Should I un-const that one?